llvm/utils/lit/tests/time-tests.py

```diff
@@ -0,0 +1,18 @@
+## Check that --skip-test-time-recording skips .lit_test_times.txt recording.
+
+# RUN: %{lit-no-order-opt} --skip-test-time-recording %{inputs}/time-tests
+# RUN: not ls %{inputs}/time-tests/.lit_test_times.txt > %t.out 2>&1
+# RUN: FileCheck --check-prefix=CHECK-NOFILE < %t.out %s
+
+## Check that --time-tests generates a printed histogram
+
+# RUN: %{lit-no-order-opt} --time-tests %{inputs}/time-tests > %t.out
+# RUN: FileCheck < %t.out %s
+# RUN: rm %{inputs}/time-tests/.lit_test_times.txt
+
+# CHECK-NOFILE: cannot access 'Inputs/time-tests/.lit_test_times.txt': No such file or directory
```

Comment:
Is there some mechanism in place that guarantees English locale?

Reply:
Could you explain what you mean in detail? Not sure I'm following here.

Reply:
I'm not sure, which command is supposed to output this line (rm/filecheck/the file redirect?). If this system output is localized, the check will fail if locale is not set to something like LANG=EN_us

Reply:
Not locale related as such, but `No such file or directory` is platform-specific and this will fail on other platforms. In other parts of llvm, we have %errc_... substitutions to allow specifying the message via the FileCheck command-line. I don't know if those are available here?

Reply:
> Not locale related as such, but No such file or directory is platform-specific and this will fail on other platforms.

Would a more reliable change be to verify that the `ls` command failed? I can't think of any other reasons why the `ls` would fail unless the file isn't found. 

> In other parts of llvm, we have %errc_... substitutions to allow specifying the message via the FileCheck command-line.

I don't think we have that here. Lit's own test suite isn't as comprehensive as other test suites. We could add that functionality, but I'm not sure it's worth for this one specific use case.